src/component/AddMovieComponent.jsx

```diff
@@ -41,7 +58,11 @@ export const AddMovieComponent = ({
                         className="mb-2"
                         required
                         name="cover_image"
-                        value={newMovie.cover_image}
+                        value={omdbMoves?.map((movie) => {
+                            if (inputTitle == movie.Title) {
+                                return movie.Poster
+                            }
+                        })}
```

Comment:
There is no need to do this, it can stay as `value={newMovie.cover_image}`, this will work fine, becuase you are already setting the newMovie.cover_image in the onClick function for OMDB movies.
Also, with `omdbMovies.map...` the user can't type/paste the URL manually